parachain/src/wasm_executor/validation_host.rs

```diff
@@ -54,15 +54,6 @@ struct WorkerExternalities {
 }
 
 impl Externalities for WorkerExternalities {
-	fn post_message(&mut self, message: TargetedMessage) -> Result<(), String> {
-		let mut inner = self.inner.lock();
-		inner.egress_data = <Vec::<TargetedMessage> as EncodeAppend>::append_or_new(
```

Comment:
The `egress_data` field on `WorkerExternalitiesInner` should now be unused and can be removed